scripts/merge_stubs_into_django.py

```diff
@@ -0,0 +1,59 @@
+from libcst import parse_module
+from libcst.codemod import CodemodContext
+from libcst.codemod.visitors import ApplyTypeAnnotationsVisitor
+
+from pathlib import Path
+
+def prRed(skk): print("\033[91m {}\033[00m" .format(skk))
+def prGreen(skk): print("\033[92m {}\033[00m" .format(skk))
+
+context = CodemodContext()
+visitor = ApplyTypeAnnotationsVisitor(context)
+
+stubs_dir = '../django-stubs'
```

Comment:
Please, make paths not `..` relative, but with `Path(__file__).parent` so that script could be run from other directories too.